test/integration/model/create.test.js

```diff
@@ -1205,10 +1216,12 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       expect(test).to.be.true;
     });
 
-    it('should only store the values passed in the whitelist', async function() {
-      const data = { username: 'Peter', secretValue: '42' };
+    it('should only store the values passed in the whitelist', async function() {      
+      // A unique column do not accept NULL in Db2. Unique column must have value in insert statement.
+      const data = dialect === 'db2' ? { username: 'Peter', secretValue: '42', uniqueName: 'name' } : { username: 'Peter', secretValue: '42' };
```

Comment:
Can you just use the d2b case? It shouldn't matter for the test case if uniqueName was defined or not